pkg/kubelet/managed/managed.go

```diff
@@ -88,6 +90,40 @@ func IsPodManaged(pod *v1.Pod) (bool, string, string) {
 	return false, "", ""
 }
 
+// IsPodSandboxManagedPod checks if a pod sandbox belongs to a managed pod
+// by looking for workload pinning annotations.
+func IsPodSandboxManagedPod(sandboxAnnotations map[string]string) bool {
```

Comment:
nit: can this be defined below where it's called in `IsManagedPodFromRuntimeService`

Reply:
Addressed, thanks.